components/tikv_util/src/security.rs

```diff
@@ -142,6 +147,10 @@ impl SecurityManager {
     pub fn cipher_file(&self) -> &str {
         &self.cipher_file
     }
+
+    pub fn x509_common_names(&self) -> HashSet<String> {
+        self.x509_common_names.clone()
+    }
```

Comment:
This method should return a reference and the caller should clone

Reply:
Done